packages/l/libsdl/xmake.lua

```diff
@@ -89,7 +83,11 @@ package("libsdl")
             end
 
             -- finding using sdl2-config didn't work, fallback on pkgconfig
-            return find_package("pkgconfig::sdl2", {require_version = opt.require_version})
+            if package.find_package then
+                return package.find_package("pkgconfig::sdl2", opt)
```

Comment:
`return package:find_package("..") `

it should be `:` not `.` here

Reply:
Oh yeah, my mistake.